packages/lang-server/src/token.ts

```diff
@@ -32,21 +29,14 @@ export const getToken = (fileContent: string, position: EditorPosition):
   // and cause us to not support multiline tokens
   const line = lines[position.line]
 
-  const lexer = new parser.PeekableLexer(line)
-  try {
-    while (true) {
-      const token = lexer.next()
-      const col = token.col - 1
-      if (col <= position.col && position.col < col + token.value.length) {
-        return { value: token.value, type: token.type }
-      }
-      if (col > position.col) {
-        return undefined
-      }
+  for (const token of parser.tokenizeContent(line)) {
+    const col = token.col - 1
+    if (col <= position.col && position.col < col + token.value.length) {
+      return { value: token.value, type: token.type }
+    }
+    if (col > position.col) {
+      return undefined
```

Comment:
```suggestion
  const tokenContains = (column: number) => (token: Token): boolean => (
    token.col < column && column < token.col + token.value.length - 1
  )
  return wu.find(parser.tokenizeContent(line), tokenContains(position.col))
```
(can also inline the implementation to `tokenContains` instead of declaring it as a separate function if you like)
```suggestion
  return wu.find(
    parser.tokenizeContent(line),
    token.col < position.col && position.col < token.col + token.value.length - 1,
  )
```